test/unit/org/apache/cassandra/index/sai/utils/ArrayPostingList.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.cassandra.index.sai.utils;
+
+import com.google.common.base.MoreObjects;
+
+import org.apache.cassandra.index.sai.disk.PostingList;
+import org.apache.cassandra.index.sai.disk.v1.postings.OrdinalPostingList;
+
+//TODO Change this to use longs
+public class ArrayPostingList implements OrdinalPostingList
+{
+    private final int[] postings;
+    private int idx = 0;
+
+    public ArrayPostingList(int[] postings)
```

Comment:
There are ~50 calls to this constructor using explicit arrays creation. We could make those calls a bit briefer by using varargs here:
```suggestion
    public ArrayPostingList(int... postings)
```